src/app/page.tsx

```diff
@@ -48,7 +49,23 @@ export default function Home() {
             <OurClientButton key={item.description} {...item} />
           ))}
         </div>
-      </div>
+      </Section>
+      <Section className="flex flex-col px-12 gap-8">
+        <span className="text-gray-600 italic underline">Testimonials</span>
```

Comment:
I thought this italic/underline combo looked a bit weird 😆  no idea how to make it better just leaving this here

Reply:
😆 😆 😆 😆  I've tried to `inventar moda`, what do you think that is weirder? The italic or the underline? Just to test without it.

Reply:
I think the italic looked a bit weird tbh